### DISCUSSION TITLE
Commits under my name that I committed not attached to my account

### DISCUSSION THREAD

**Discussion by david-engelmann** [community/community#37630] [Other Feature Feedback, Questions, & Ideas] [answered]
I was looking at an insights page after a pr was merged and noticed something strange - https://github.com/doccano/doccano-client/pulse. I noticed my name was on the chart twice, one for the most recent pr and one for the pr before. Shouldn't both be associated with my user

**Comment by DenverCoder1** [comment]
Can you add a screenshot? That page does not seem to show any names when I open it. It just says things like "8 Pull requests merged by 2 people".

**Comment by airtower-luna** [comment, marked as answer]
Only some of those commits are recognized as yours, the one with the grey icon aren't, they just show up under the author name listed in the commit. The difference is the *author email* address in the commit:

* This one is recognized as yours, uses the "noreply" address provided by GitHub: https://github.com/doccano/doccano-client/commit/664231d99ba9be02e0d755735b675f7a4d782ac9.patch
* This one uses some other address: https://github.com/doccano/doccano-client/commit/faf0a5f1d4250f3306ff97e18bddfa0edb6ed94b.patch

The easiest solution is to add that email address to your GitHub account, then the commits will be recognized as yours. If you don't actually mean to use that address, fix the `user.email` setting on the computer where you made the affected commits (for future commits).

**Reply by david-engelmann** [reply]
Ops must have accidentally used the wrong git folder. Thank you for the help!
